erpnext/accounts/report/general_ledger/general_ledger.js

```diff
@@ -146,6 +146,12 @@ frappe.query_reports["General Ledger"] = {
 				return frappe.db.get_link_options('Project', txt);
 			}
 		},
+		{
+			"fieldname": "include_dims",
```

Comment:
```suggestion
			"fieldname": "include_dimensions",
```
Update everywhere, try to avoid abbreviations and shorthand if possible for better code understanding

Reply:
Got it.